### PR TITLE
Issue #1: Create the Drupal Search API project for Europa Search

### PR DESCRIPTION
**Pull request by gillesdeudon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,10 @@ env:
...
-  - composer install
-
 
 script:
-  - grumphp run
   - scripts/setup-dev-env.sh
+  - grumphp run
 
 notifications:
   email: false
```
Comment (aritomelo) [inline .travis.yml]: Missing empty line after line 18.

File: composer.json [hunk 2]
```diff
@@ -2,14 +2,40 @@
   "name":"ec-europa/search_api_europa_search",
   "type":"drupal-module",
   "description":"Provides an Europa Search backend for the Search API module.",
+  "minimum-stability": "dev",
```
Comment (aritomelo) [inline composer.json]: I am not sure if we are following any coding standard for json files, however, I'd suggest we use the same syntax across the whole file.
Example: 
On line3, there is no space between colon ":" and the asterisk that wraps the sentence. 
``` "type":"drupal-module" ```
But
On line5 It has space 
``` "minimum-stability": "dev", ```

Comment (gillesdeudon) [reply composer.json]: Ok. Adaptation done.

File: .travis.yml (hunk 1 above)
Comment (gillesdeudon) [reply .travis.yml]: Ok. Adaptation done.

File: composer.json [hunk 3]
```diff
@@ -1,15 +1,41 @@
 {
-  "name":"ec-europa/search_api_europa_search",
-  "type":"drupal-module",
-  "description":"Provides an Europa Search backend for the Search API module.",
+  "name": "ec-europa/search_api_europa_search",
+  "type": "drupal-module",
+  "description": "Provides an Europa Search backend for the Search API module.",
+  "minimum-stability": "dev",
```
Comment (voidtek) [inline composer.json]: If you want used one dev requirement, is recommended to use stability flags.
Example:
```
"minimum-stability": "stable",
"require": {
  ...
  "ec-europa/oe-europa-search-client": "dev-nept-1558-issue17@dev"
},
```

See more information here: https://igor.io/2013/02/07/composer-stability-flags.html

File: composer.lock [hunk 4]
```diff
@@ -0,0 +1,5055 @@
+{
```
Comment (voidtek) [inline composer.lock]: This file is not mandatory on the repo.

**Pull request merged**